src/expression.ts

```diff
@@ -200,13 +205,13 @@ export class ElementAccessExpr extends BaseExpr<"ElementAccessExpr"> {
 export const isArgument = typeGuard("Argument");
 
 export class Argument extends BaseExpr<"Argument", CallExpr | NewExpr> {
-  constructor(readonly expr: Expr, readonly name?: string) {
+  constructor(readonly expr?: Expr, readonly name?: string) {
```

Comment:
since when did expr become optional?

Reply:
This is because of the signature processing for CallExprs. 

All arguments in the signature are added to the array, but were creating an undefined literal expression for all unprovided arguments. So I changed it to instead provide `undefined` for unprovided arguments. It maintains the named argument still.